src/components/Video.js

```diff
@@ -14,7 +15,8 @@ class Video extends React.Component {
 	}
 
 	loadPlayer () {
-		require(['react-player'], (mod) => this.setState({ VideoComponent: mod.default }));
+		window.React = React;
+		loadScript('https://unpkg.com/react-player@1.11.0/dist/ReactPlayer.js').then(() => this.setState({ loaded: true }));
```

Comment:
Do we really need to depend on external URL? Can't load it via npm?

Reply:
There are major issues making libraries load additional code asynchronously. Loading it using npm like it was done before (`require(['react-player'])`) introduces code-split chunks to the library bundled separately, but webpack doesn't know where to load these async assets from when they need to be loaded.

And if **sokra** says it's not yet supported — https://github.com/webpack/webpack/issues/6818#issuecomment-377225847 — then it's not yet supported.

I shall investigate our options here though, but can't promise anything.